jsonobject/containers.c

```diff
@@ -1,4 +1,4 @@
-/* Generated by Cython 0.29.1 */
+/* Generated by Cython 0.29.6 */
```

Comment:
Not sure if this is a problem, but I noticed that this was built with a different version of Cython. If you see an error with tests or anything that would be something to look at.